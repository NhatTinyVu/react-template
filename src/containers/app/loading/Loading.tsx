import React from 'react';
import { Spin, Icon } from 'antd';
import styles from './Loading.scss';

const Loading: React.FC = () => (
  <Spin
    className={styles.container}
    indicator={<Icon type="loading" className={styles.icon} spin />}
  />
);

export default Loading;
