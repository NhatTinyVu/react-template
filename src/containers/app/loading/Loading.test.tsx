import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import Loading from '.';

it('renders correctly', (): void => {
  const wrapper = shallow(<Loading />);
  expect(toJSON(wrapper)).toMatchSnapshot();
});
