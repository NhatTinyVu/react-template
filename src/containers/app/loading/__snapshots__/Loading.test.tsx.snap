// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Spin
  className="container"
  indicator={
    <Icon
      className="icon"
      spin={true}
      type="loading"
    />
  }
  size="default"
  spinning={true}
  wrapperClassName=""
/>
`;
